backend/copr_backend/pulp.py

```diff
@@ -181,12 +180,16 @@ def create_content(self, path, labels):
         Create content for a given artifact
         https://docs.pulpproject.org/pulp_rpm/restapi.html#tag/Content:-Packages/operation/content_rpm_packages_create
         """
-        url = self.url("api/v3/content/rpm/packages/")
+        url = self.url("api/v3/content/rpm/rpmpackages/")
         with open(path, "rb") as fp:
-            data = {"pulp_labels": json.dumps(labels)}
             files = {"file": fp}
-            return requests.post(
-                url, data=data, files=files, **self.request_params)
+            package =  requests.post(
+                url, files=files, **self.request_params)
+        if package.ok:
+            package_href = package.json()["pulp_href"]
+            for key, val in labels.items():
+                self.set_label(package_href, key, val)
```

Comment:
Should we wait till the set_label task finishes?

Reply:
@praiskup set_label is not a task.